styles/components/_containers.scss

```diff
@@ -364,6 +370,45 @@
   }
 }
 
+.arrowBoxTipWrap {
```

Comment:
Create a re-usable tooltip alternative to the standard one we have in place. The main benefit of this one is that it allows you to put full markup (as opposed to plain text) in the tooltip. A usage example is:

```
    <div class="arrowBoxTipWrap">
      <span>Hover over me for fun times!</span> // the tooltip trigger
      <div class="arrowBoxCenteredTop clrBr clrP"> // the content that appears on hover
         <p>It was the best of times. It was the worst of times.</p>
      </div>
    </div>
```

For the content, you currently have two options that the determine the positioning of the tip (arrowBoxCenteredTop & arrowBoxRightTop). We could add more (e.g. options that position the tip on top of the trigger, etc...). Although, I may need to slightly tweak the markup to properly support that (I'll keep you in the loop if I do it).

The main point is that this should be a re-usable component, so we shouldn't have to have a bunch of custom one off tooltips implemented around the app. Currently, you could choose between the simple plain text (`data-tip`) one of this one. Eventually, we can even consider eliminating the simple one, since this one can also do the same thing with only slightly more markup.

Reply:
Good work here.